app/src/main/java/com/jalasoft/routesapp/data/model/remote/User.kt

```diff
@@ -0,0 +1,12 @@
+package com.jalasoft.routesapp.data.model.remote
+
+data class User(
+    val id: String? = null,
+    val name: String? = null,
+    val email: String? = null,
+    val phoneNumber: String? = null,
+    val type: Int? = null,
+    val typeLogin: Int? = null,
+    val createdAt: Double? = null,
+    val updatedAt: Double? = null
```

Comment:
Does the date has decimals? or why double instead of long

Reply:
because in ios it is saving with decimals so to don't get issues